src/visual/Visualizer.js

```diff
@@ -86,6 +86,20 @@ export default class Visualizer {
     this._timer = null;
     this._step = null;
     this.clear();
+
+    this.paperDefaultValidateConnectionMethod = this.paper.options.validateConnection;
+
+    this.paper.options.validateConnection = (cellViewS, magnetS, cellViewT, magnetT, end, linkView) => {
+      const args = [cellViewS, magnetS, cellViewT, magnetT, end, linkView];
+
+      if (this.paperDefaultValidateConnectionMethod.call(this.paper, ...args)) {
```

Comment:
I am a little bit concerned about the usage of the **spread syntax** here since there is an _apply_ method which does not require dots.